paper/src/main/java/com/denizenscript/denizen/paper/events/UnknownCommandScriptEvent.java

```diff
@@ -53,60 +54,37 @@ public class UnknownCommandScriptEvent extends BukkitScriptEvent implements List
 
     public UnknownCommandScriptEvent() {
         registerCouldMatcher("command unknown");
+        this.<UnknownCommandScriptEvent, ElementTag>registerDetermination(null, ElementTag.class, (evt, context, text) -> {
+            evt.event.message(PaperModule.parseFormattedText(text.toString(), ChatColor.WHITE));
+        });
+        this.<UnknownCommandScriptEvent>registerTextDetermination("none", (evt) -> {
+            evt.event.message(null);
+        });
     }
 
     public UnknownCommandEvent event;
     public String command;
     public String rawArgs;
     public String sourceType;
 
-    @Override
-    public boolean applyDetermination(ScriptPath path, ObjectTag determinationObj) {
-        if (determinationObj instanceof ElementTag) {
-            String determination = determinationObj.toString();
-            if (CoreUtilities.equalsIgnoreCase(determination, "none")) {
-                event.setMessage(null);
-            }
-            else {
-                event.setMessage(determination);
-            }
-            return true;
-        }
-        return super.applyDetermination(path, determinationObj);
-    }
-
     @Override
     public ScriptEntryData getScriptEntryData() {
         return new BukkitScriptEntryData(event.getSender() instanceof Player ? new PlayerTag((Player) event.getSender()) : null, null);

```

Comment:
I know you didnt edit this, but can update to pattern matching as well while you are at it